genomic_data_service/region_indexer.py

```diff
@@ -113,6 +120,10 @@
     "--local", action='store_true',
     help = "Index a small number of files for local install")
 
+parser.add_argument(
+    "--assembly", default=SUPPORTED_ASSEMBLIES, nargs='*',
```

Comment:
If we expect multiple (as it seems from the default) this should probably be called `--assemblies`.